bindinfo/capture_test.go

```diff
@@ -555,6 +555,62 @@ func TestIssue25505(t *testing.T) {
 	}
 }
 
+func TestCaptureUserFilter(t *testing.T) {
```

Comment:
I'll add more test cases when supporting regular-expression filters in the next few PRs.